_posts/2024-12-05-vw-symposium.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Code"
```

Comment:
Ich gebe den Artikel zwar trotzdem schon einmal frei, aber ist "Code" nicht ein recht kurzer, zu allgemeiner Titel?

Reply:
Ich vermute auch, dass das "Code for Science, oder: Bessere Forschungssoftware durch bessere Forschungssoftwarekompetenzen" heißen sollte

Reply:
wenn man mal "eben schnell" etwas erledigen will :( Danke für den Hinweis.